actionpack/test/controller/support/custom_assertions.rb

```diff
@@ -0,0 +1,3 @@
+def assert_filtered_out(params, key)
+  assert !params.has_key?(key), "key #{key.inspect} has not been filtered out"
+end
```

Comment:
I'm hesitant to do something at all. I'd rather not extract helpers to early. It's much harder to remove unused code once it's been generalized. I'd probably keep the current code until we see more repetitions.